addons/web/static/src/core/notebook/notebook.scss

```diff
@@ -49,7 +49,12 @@
             outline: none;
         }
 
-        &:hover {
+        &:focus-visible, &.active:focus-visible {
+            box-shadow: inset 0 0 0 $border-width * 3 rgba($o-brand-odoo, 0.2);
```

Comment:
- The focus-visible state is better on Enterprise, great 👍  But it seems to still be broken on Community, could you have a look at it ?

| COM | ENT |
|--------|--------|
| <img width="352" alt="Capture d’écran 2025-06-11 à 14 18 14" src="https://github.com/user-attachments/assets/ec136f96-ff9a-4039-bd92-ff0925f9be9d" /> | <img width="359" alt="Capture d’écran 2025-06-11 à 14 18 25" src="https://github.com/user-attachments/assets/addfe952-55a2-4cb1-afef-27bd340ebe38" /> | 

- Also, I think you could reuse values of the initial "Focus Ring" to reproduce this "inset" version.
https://github.com/odoo/odoo/blob/master/addons/web/static/lib/bootstrap/scss/_variables.scss#L569-L573